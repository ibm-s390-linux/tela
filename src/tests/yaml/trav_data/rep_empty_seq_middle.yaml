- seq1
- replace
- seq2
