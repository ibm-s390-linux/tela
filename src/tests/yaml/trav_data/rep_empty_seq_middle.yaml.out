Before (root=non-null):
==================================
  - seq1
  - replace
  - seq2
==================================

Callback:
==================================
seq1                                    : -
seq1/                                   :   seq1
replace                                 : -
*** Replacing node
seq2                                    : -
seq2/                                   :   seq2
==================================

After (root=non-null):
==================================
  - seq1
  replacement
  - seq2
==================================
